presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -335,4 +337,24 @@ default void revokeTablePrivileges(ConnectorSession session, SchemaTableName tab
     {
         throw new PrestoException(NOT_SUPPORTED, "This connector does not support revokes");
     }
+
+    default TableIdentity getTableIdentity(ConnectorTableHandle connectorTableHandle)
+    {
+        throw new PrestoException(NOT_SUPPORTED, "This connector has not implemented TableIdentity");
+    }
+
+    default Map<String, ColumnIdentity> getColumnIdentities(ConnectorTableHandle connectorTableHandle)
```

Comment:
Why not just have `ColumnIdentity getColumnIdentities(ColumnHandle columnHandle)`?